cogs/exceptions.py

```diff
@@ -74,18 +75,21 @@ async def on_command_error(self, ctx: Context, error: commands.CommandError) ->
             None.
         """
 
+        if ctx.command is None and isinstance(error, commands.CommandNotFound) and ctx.guild is not None:
```

Comment:
`if ctx.command is None and ctx.guild is not None and isinstance(error, commands.CommandNotFound):`